source/whats-new.txt

```diff
@@ -36,6 +37,18 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.5:
+
+What's New in 5.5
+-----------------
+
+New features of the 5.5 {+driver-short+} release include:
+
+- The driver now accurately detects Function-as-a-Service (FaaS)
+  environments in AWS.
```

Comment:
Issue:
This seems like it fixes a bug, but that bug hasn't clearly been described. From reading this, I'm not sure what it means to more accurately detect a FaaS environment in AWS.

Suggestion:
Provide more detail on what the behavior was previously and what it more accurately detects.